Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -387,6 +387,7 @@ void WebFrame::loadDidCommitInAnotherProcess(std::optional<WebCore::LayerHosting
         frame->setOpener(newFrame.ptr());
 
     newFrame->tree().setSpecifiedName(localFrame->tree().specifiedName());
+    newFrame->setOwnerElement(ownerElement.get());
```

Comment:
Or we could remove the ownerElement argument from `createSubframeWithContentsInAnotherProcess`.

Reply:
And the test could be a layout test.

Reply:
Yeah, this call to setOwnerElement is not needed but the other is.  We can't pass it in to the constructor because the owner element shouldn't know about a provisional frame until it commits.